recipes/ratarmount/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ratarmount" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ratarmount-{{ version }}.tar.gz
+  sha256: 2bfb270bb91a326c72c24eca82f7b79a8fd077904eb9ce476fe46da23a1f9073
+
+build:
+  entry_points:
+    - ratarmount=ratarmount:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=46.4.0
+    - wheel
+    - pip
+  run:
+    - python >=3.6
+    - ratarmountcore >=0.4.0,<0.5.dev0
+    - fusepy
+    - indexed_bzip2 >=1.3.1,<2.0
+    - indexed_gzip >=1.6.3,<2.0
+    - indexed_zstd >=1.3.1,<2.0
+    - dataclasses
+    - python-xz >=0.4.0,<0.5.dev0
+    - rarfile >=4.0,<5.dev0
+
+test:
```

Comment:
Please still include the import test, even if it fails on OSX and Win. As it is noarch, it will only be built on Linux on the feedstock.